packages/apps/api/src/server-lambda.ts

```diff
@@ -1,17 +1,10 @@
 import { APIGatewayProxyEvent, Context } from 'aws-lambda';
 import * as awsServerlessExpress from 'aws-serverless-express';
 import { Server } from 'http';
-import { HttpSierraClient, SierraClient } from '@weco/sierra-client';
 import { HttpAuth0Client, Auth0Client } from '@weco/auth0-client';
 import { createApplication } from './app';
 import HttpEmailClient, { EmailClient } from './utils/EmailClient';
 
-const sierraClient: SierraClient = new HttpSierraClient(
-  process.env.SIERRA_API_ROOT!,
```

Comment:
We need to remember to remove these from the Terraform definitions.

Reply:
Good point, I'll put that in this PR